### PR TITLE
$ is now a method of utils

### PR DESCRIPTION
**Pull request by rittels**

### CODE REVIEW

File: examples/07-multiple-files-plugin/soundingGraph.mjs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import overlays from '@windy/overlays';
 import rs from '@windy/rootScope';
 import store from '@windy/store';
-import $ from '@windy/$';
 import _ from '@windy/utils';
+const {$} = _;
```
Comment (mdorda) [inline examples/07-multiple-files-plugin/soundingGraph.mjs:5]: Please, use prettier to keep formatting consistent. Also, imports first, declarations below.

File: examples/11-mobile-demo/plugin.html [hunk 2]
```diff
@@ -72,7 +72,8 @@ <h2>This plugin is:</h2>
     <div class="calendar-backdrop"></div>
 
     <script>
-        import $ from '@windy/$';
+        import _ from '@windy/utils';
+        const {$} = _;
```
Comment (mdorda) [inline examples/11-mobile-demo/plugin.html:76]: Please, use prettier to keep formatting consistent. Also, imports first, declarations below.

File: examples/11-mobile-demo/plugin.html [hunk 3]
```diff
@@ -85,13 +86,8 @@ <h2>This plugin is:</h2>
...
         if (bgVals.length == 3) bgVals.push(1);
-        $(
-            '.calendar-backdrop',
-            this.el,
-        ).style.backgroundImage = `linear-gradient(rgba(${bgVals.slice(
-            0,
-            3,
-        )},0), rgba(${bgVals}) 60%)`;
+        $( '.calendar-backdrop', this.el )
+            .style.backgroundImage = `linear-gradient(rgba(${bgVals.slice(0, 3)},0), rgba(${bgVals}) 60%)`;
```
Comment (mdorda) [inline examples/11-mobile-demo/plugin.html:90]: Is there any change?

File: dist/plugin.js [hunk 4]
```diff
@@ -15,116 +15,37 @@ W.loadPlugin(
     "url": "git+https://github.com/windycom/windy-plugins"
```
Comment (mdorda) [inline dist/plugin.js:15]: Why so many changes in this file? Is it on purpose?

**Review by mdorda — CHANGES_REQUESTED** [review]

**Comment by rittels** [comment]
new PR

**Pull request closed without merging**